set.mm

```diff
@@ -154502,15 +154516,24 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
     ZABEDZUIFDZAGEDHAIJBKJZUKUHUJLMQQABBNOUGGAEPRUJHUAZULFDHHFDHUIULUIULFSSUBHH
     TTUCUDUEUF $.
 
-  $( A positive integer is less than or equal to its square.  Note that this
-     holds for all integers.  (Contributed by NM, 15-Sep-1999.)  (Revised by
+  $( A positive integer is less than or equal to its square.  For general
+     integers, see ~ zzlesq .  (Contributed by NM, 15-Sep-1999.)  (Revised by
```

Comment:
integers in general

Reply:
I think "for general integers" is more common and idiomatic.